src/main/java/io/vertx/ext/mail/MailConfig.java

```diff
@@ -942,6 +957,50 @@ public MailConfig setMultiPartOnly(boolean multiPartOnly) {
     return this;
   }
 
+  /**
+   * @return the connection pool cleaner period in ms.
+   */
+  public int getPoolCleanerPeriod() {
+    return poolCleanerPeriod;
+  }
+
+  /**
+   * Set the connection pool cleaner period in milli seconds, a non positive value disables expiration checks and connections
+   * will remain in the pool until they are closed.
+   *
+   * @param poolCleanerPeriod the pool cleaner period
+   * @return a reference to this, so the API can be used fluently
+   */
+  public MailConfig setPoolCleanerPeriod(int poolCleanerPeriod) {
```

Comment:
I think like other options it would be appropriate to have also a poolCleanerPeriodUnit

Reply:
done